tools/barrnap/barrnap.xml

```diff
@@ -52,24 +52,24 @@
     </outputs>
 
     <tests>
-        <test><!-- Test 1 -->
+        <test expect_num_outputs="1"><!-- Test 1 -->
```

Comment:
Guess we can remove a linter rule from the skip file. 

We should bump and deploy this one (so maybe better put in a separate PR)?

Reply:
Not sure a deployment is needed here? I would not deploy for a stricter test. Please feel free to merge and take it from here.